cinch/roles/jenkins_master/tasks/ensure_up.yml

```diff
@@ -1,16 +1,6 @@
-- name: set listening port
-  set_fact:
-    _listening_port: "{{ https_enabled | ternary(443, 80) }}"
-  when: ansible_connection != 'docker'
-
-- name: check for running in a container
-  set_fact:
-    _listening_port: "{{ https_enabled | ternary(8443, 8080) }}"
-  when: ansible_connection == 'docker'
-
 - name: construct Jenkins URL
   set_fact:
-    _jenkins_url: "http{{ https_enabled | ternary('s', '') }}://localhost:{{ _listening_port }}"
+    _jenkins_url: "http{{ https_enabled | ternary('s', '') }}://{{ inventory_hostname }}"
```

Comment:
Why have you made these changes to this file? You've removed logic that was necessary to get Docker working properly, and you've changed to using "inventory_hostname" instead of localhost, which can easily lead to an invalid URL as inventory_hostname can be set to something like, "my_computer_of_awesomeness".